cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file is for configuration for cypress. It is auto-generated by running `cypress open` command.